# If min angle > max angle, the scan will be executed behind the vehicle, 
# .. ie the scan will include 180 degrees
float32 min_angle_deg
float32 max_angle_deg
float32 range